python/setup.py

```diff
@@ -137,7 +137,7 @@ def find_version(*filepath):
 
 
 requires = [
```

Comment:
Should this actually be ‘1.14.5’?

Reply:
No, minor versions are compatible, so if the user for example has 1.14.3 installed that's fine. We are just building against the latest minor version released.